homeassistant/components/flexit_bacnet/climate.py

```diff
@@ -83,6 +84,22 @@ async def async_update(self) -> None:
         """Refresh unit state."""
         await self.device.update()
 
+    @property
+    def extra_state_attributes(self) -> dict[str, Any]:
+        """Return device specific state attributes."""
+        return {
+            "air_filter_exchange_interval": self.device.air_filter_exchange_interval,
+            "air_temp_setpoint_away": self.device.air_temp_setpoint_away,
+            "air_temp_setpoint_home": self.device.air_temp_setpoint_home,
```

Comment:
Should these be sensors or number entities instead of just adding the as attributes here?

Reply:
Yeah, these should probably be numbers. Will remove them from attributes.